assets/kopijs.js

```diff
@@ -20,69 +20,107 @@
   for (var i=0, l=$kopis.length; i<l; i++){
     var $kopi = $kopis[i];
     var name = $kopi.textContent;
+    makeKopi($kopi, name);
+  }
 
-    // Magic happens from Kopi.js
-    var ingredients = Kopi.parse(name);
+  function makeKopi($kopi, name, ingredients){
 
-    // Configure the internal ingredients of the Kopi
-    var internalIngredients = 'water condensed_milk evaporated_milk coffee'.split(' ');
-    var internalIngredientsHTML = internalIngredients.map(function(ingredient){
-      var value = ingredients[ingredient];
-      if (!value) return;
-      var height = value * 100 + '%';
-      return '<div class="' + ingredient + '" style="height: ' + height + '"></div>';
-    }).join('');
+      // Magic happens from Kopi.js
+      if (!ingredients) {
+          var ingredients = Kopi.parse(name);
```

Comment:
I think no need `var` anymore since the variable is already "declared" from the arguments. Including the `else` statement below.